index.html

```diff
@@ -1,199 +1,326 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Page</title>
```

Comment:
We changed it to Portfolio Page

Reply:
Very detailed report 👍🏽 but the `Portfolio` project requirement for the title is that It adequately and briefly describes the content of the page. For example:

### Poor titles:
```
Microverse project | Your portfolio.
```
or
```
Portifolio project
```

### Better page titles:
```
Mark's portfolio
```
or
```
Mark full-stack webdeveloper
```